generichttp/motion/mover.go

```diff
@@ -25,13 +25,23 @@ type Mover interface {
 	Home(string) error
 }
 
+type HomeQuerier interface {
+	// GetHome gets if an axis is homed
+	GetHomed(string) (bool, error)
```

Comment:
I suggest `Homed()` - Get and Set should come in pairs, but there is no `SetHomed()`